pkg/oci/load_balancer_util.go

```diff
@@ -29,8 +29,14 @@ import (
 )
 
 const (
-	sslCertificateFileName = "tls.crt"
-	sslPrivateKeyFileName  = "tls.key"
+	// SSLCAFileName is a key name for ca data in the secrets config
```

Comment:
As docstrings are expected to be sentences (for nice godocs) they should end with a full-stop.